its/projects/CSharp.SDK.7.0/AspNetCoreMvc/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace AspNetCoreMvc.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
+    {
+        private static readonly string[] Summaries = new[]
+        {
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+    };
```

Comment:
The indentation is wrong here. And also do we really need all these values for the summaries? Less is more :)